serialization/json_ld/examples/spdx_document4.json

```diff
@@ -0,0 +1,59 @@
+{
+  "@context": [
+    "https://spdx.github.io/spdx-3-model/rdf/context.json",
+    {
+      "myNamespace": "https://some.namespace#",
+      "spdxDev": "https://spdx.dev/elements/3F26391C#",
+      "spdxLicenses": "https://spdx.org/licenses/"
+    }
+  ],
+  "element": [
+    {
+      "type": "SpdxDocument",
+      "spdxId": "myNamespace:spdxdocument159",
+      "creationInfo": {
+        "type": "CreationInfo",
+        "specVersion": "3.0.0",
+        "created": "2022-12-01T00:00:00",
+        "createdBy": ["spdxDev:spdx-dev"],
+        "profile": ["core", "software"],
+        "dataLicense": "spdxLicenses:CC0-1.0"
+      },
+      "name": "Doc 159 - two File elements",
+      "element": [
+        "myNamespace:SPDXRef-Package",
+        "myNamespace:File1",
+        "spdxDev:spdx-spec-v2.3",
+        "myNamespace:relationship1"
+      ],
+      "rootElement": [
+        "myNamespace:SPDXRef-Package"
+      ]
+    },
+    {
+      "type": "Package",
+      "spdxId": "myNamespace:SPDXRef-Package",
+      "name": "packageName"
+    },
+    {
+      "type": "File",
+      "spdxId": "myNamespace:file1",
+      "name": "file1"
+    },
+    {
+      "type": "File",
+      "spdxId": "myNamespace:file2",
+      "name": "file2"
+    },
+    {
+      "type": "Relationship",
+      "spdxId": "myNamespace:relationship1",
+      "from": "myNamespace:SPDXRef-Package",
+      "to": [
+        "myNamespace:file1",
+        "myNamespace:file2"
+      ],
+      "relationshipType": "contains"
```

Comment:
This contradicts the last bullet in `rdf.md`.
Unless the context in #389 will be extended to have all enumerated values.

Reply:
Just a note - in the SPDX 2.X spec, we used a convention of prefixing the enumeration value with the type for RDF values to make sure they are unique across the entire namespace (e.g. `relationshipType_contains`).  Without such a convention, we are likely to have collisions in the enumeration values.

Reply:
@goneall the `rdf.md` file already says:

>  IRIs for an enumerated value are of the form: `http://spdx.org/rdf/v3/{Namespacename}/{Classname}/{Valuename}`

Reply:
Ah, sorry, @armintaenzertng , I only quickly searched for `"contains"` in the context file and did not see anything, so I assumed it was not handled. Using `@vocab` is perfectly fine as a first step -- or even a final one.